### ISSUE TITLE
Changing the tabs.title.format placeholders

### ISSUE THREAD

**Issue by karma-riuk** [qutebrowser/qutebrowser#5689]
Hello,

I've very recently come across your project and it's pretty damn cool.
Unfortunately, in my journey to customize the interface, I've encountered something that I don't understand how (if it's even possible) to change the placeholders given in the tabs.title.format.

I'm specifically aiming for the "{audio}". The way to signal that a tab is muted is not quite my style. I'd rather prefer a simple mute character instead of a "[M]". The mute character that I thought about isn't anything fancy, just the unicode character for it (I don't know to show it on this post, so [here](https://www.fileformat.info/info/unicode/char/1f507/index.htm) is a reference to it).

Anyways, nothing too important... I just wanted to know if there were an option the change those placeholders.

Have a good day :)

**Comment by toofar** [comment]
It is not a first class configurable thing but there is a way to do it documented here: https://github.com/qutebrowser/qutebrowser/pull/4089

I would propose something that doesn't make qutebrowser fail to load your config when that implementation detail changes (hopefully message can be used at this point in the init process):
```python
from qutebrowser.api import message
try:
    from qutebrowser.mainwindow import tabwidget
    tabwidget.TabWidget.MUTE_STRING = "I AM NOT LOUD: "
    tabwidget.TabWidget.AUDIBLE_STRING = "I AM VERY LOUD: "
except ImportError, AttributeError as err:
    message.error(f"Failed to change audible indicators {err}")
```

**Comment by karma-riuk** [comment]
Oh yeah, that's exactly what I needed.
Sorry for the duplicate then, before posting, I tried to look through the issues but it was hard to find anything.
Thanks a lot! 

Note: the code snippet you gave me is really nice, but the is a small error, when I copied it into my `config.py` and ran `config-source` in the browser, I had an error message telling me there was a syntax error. In fact, after some digging, I noticed that the way to catch the `ImportError` and `AttributeError` is missing the brackets, in order to be catched as one error.

So the line 
```{python}
    except ImportError, AttributeError as err:
```
should become
```{python}
    except (ImportError, AttributeError) as err:
```
in order not to have syntax errrors.

Have a nice day :)
